v2/azure-arm.yaml

```diff
@@ -449,6 +449,11 @@ exportFilters:
     version: v*api20210301
     name: RedisEnterpriseDatabase
     because: "including redis enterprise resources"
+  - action: include-transitive
+    group: containerregistry
+    version: v*api20190501
```

Comment:
Can you try using the new 2021-09-01 API instead? It's significantly newer.

To do this we will have to update our submodule so that we can see it. I've just opened a PR to do that: #2075

Once that PR merges you can merge with it (or rebase onto it) and should be able to use `20210901` here (and below) instead.

Reply:
That PR merged so you should be able to try the newer API version

Reply:
I've just update api on newer 2021-09-01 and now I have this error:

```
E0209 10:26:50.957979   23472 gen_kustomize.go:111] Error during code generation:
failed during pipeline stage 23/56: Verify there are no ErroredType's containing errors: "github.com/Azure/azure-service-operator/v2/api/containerregistry/v1alpha1api20210901/Registry" has property "Status" with errors: "missing status information for github.com/Azure/azure-service-operator/v2/api/containerregistry/v1alpha1api20210901/Registries"
```

IDK what I'am doing wrong. Can you help me and explain the reason of this error?
May be this related to that Registry ARM resource [does not have status property](https://github.com/Azure/azure-resource-manager-schemas/blob/60b772a297ef5dff7feb57761dd26557bd600bf3/schemas/2021-09-01/Microsoft.ContainerRegistry.json#L9)?

Reply:
Ah, this is my fault! I updated one of our submodules but not the other. Here's a PR that fixes it: #2079

I've also confirmed locally that with that PR merged your changes here should work.
You will need to replace your section on line 951 with this, I believe:
```
  containerregistry:
    2021-09-01:
      IdentityProperties:
        PrincipalId:
          $armReference: false
```

Mark that as `$armReference: false` because that field isn't an ARM reference (ARM ID) it's the principal ID of the identity in AAD.

Reply:
Thanks!